breakers.gemspec

```diff
@@ -0,0 +1,48 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'breakers/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'breakers'
+  spec.version       = Breakers::VERSION
+  spec.authors       = ['Aubrey Holland']
+  spec.email         = ['aubrey@adhocteam.us']
+
+  spec.summary       = 'Handle outages to backend systems with a Faraday middleware'
+  spec.description   = 'This is a Faraday middleware that detects backend outages and reacts to them'
+  spec.homepage      = ''
+  spec.license       = 'MIT'
+
+  # Prevent pushing this gem to RubyGems.org. To allow pushes either set the 'allowed_push_host'
+  # to allow pushing to a single host or delete this section to allow pushing to any host.
+  if spec.respond_to?(:metadata)
+    spec.metadata['allowed_push_host'] = "TODO: Set to 'http://mygemserver.com'"
+  else
+    raise 'RubyGems 2.0 or newer is required to protect against public gem pushes.'
+  end
+
+  spec.files = `git ls-files -z`.split("\x0").reject do |f|
+    f.match(%r{^(test|spec|features)/})
+  end
+  spec.bindir        = 'exe'
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+
+  spec.add_dependency 'faraday'
+  spec.add_dependency 'multi_json'
```

Comment:
It's not assuming the response is in JSON, it uses JSON to serialize data about the outages in Redis. The use of multi_json is just a performance improvement here, since it'll choose the fastest JSON serializer available.